sklearn/feature_extraction/tests/test_text.py

```diff
@@ -51,7 +51,7 @@
 NOTJUNK_FOOD_DOCS = (
     "the salad celeri copyright",
     "the salad salad sparkling water copyright",
-    "the the celeri celeri copyright",
+    "the celeri celeri copyright",
```

Comment:
This is not a typo. The [tests are failing](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=41598&view=logs&jobId=dde5042c-7464-5d47-9507-31bdd2ee0a3a&j=dde5042c-7464-5d47-9507-31bdd2ee0a3a&t=4bd2dad8-62b3-5bf9-08a5-a9880c530c94), because they expect "the" to be repeated here. Can you revert this change?